doc/compiler/jitserver/Build.md

```diff
@@ -128,6 +118,26 @@ bash get_source.sh -openj9-repo=https://github.com/<Your GitHub UserID>/openj9.g
 ```
 See https://www.eclipse.org/openj9/oj9_build.html for more detail. The only difference is you need to check out the JITServer code instead of upstream OpenJ9.
```

Comment:
Let's remove "The only difference is you need to check out the JITServer code instead of upstream OpenJ9."
This probably is a leftover since we had a different branch.